prime-router/build.gradle.kts

```diff
@@ -112,6 +112,7 @@ val env = mutableMapOf<String, Any>(
     "VAULT_API_ADDR" to "http://localhost:8200",
     "SFTP_HOST_OVERRIDE" to "localhost",
     "SFTP_PORT_OVERRIDE" to "2222",
+    "ETOR_TI_baseurl" to "http://host.docker.internal:8080",
```

Comment:
Also, I think TI-integration should be optional. That means our startup scripts should not automatically spin up an instance of TI (which it doesn't) but that also means not including `ETOR_TI_baseurl` in gradle or docker-compose. TI functions should assert the ETOR_TI integration is properly configured before doing its logic.

Reply:
We've added the endpoints to the .yml API files.  Would it be possible to leave the environment variable in build.gradle and the docker-compose files? Currently it shouldn't impact any of the report stream developers and it would prevent the intermediary developers from having to manually add this environment variable every time we need to to do development or testing on one of these two endpoints.  Additionally we would avoid running a check for the environment variables existence in each of the deployed API calls for a value that absolutely should be there.

Reply:
The two bullet points for documentation have been addressed

Reply:
In the spirit of this being an open source project, I tend to agree with arnej that we shouldn't add new code/configuration that won't work for everyone.  Let me know if you want some help brainstorming how to make it easier to spin up locally, but a few things you could do:

- new gradle task that sets up the environment
- modify the local.env and just leave it there

I do think we should be checking for the service being available.  As written a new developer ramping up here would hit the endpoint and after a timeout get a strange error message